lib/router/search.js

```diff
@@ -1,18 +1,15 @@
 const { get } = require('lodash');
 const { Router } = require('express');
 const { ElasticError, NotFoundError } = require('../errors');
-const { createESClient } = require('../elasticsearch');
 const search = require('../search');
 
 module.exports = (settings) => {
   const app = Router();
 
-  const client = createESClient(settings.es);
-
   app.use((req, res, next) => {
-    return Promise.resolve(client)
-      .then(esClient => {
-        req.search = search(esClient);
+    return Promise.resolve(settings.esClient)
+      .then(client => {
+        req.search = search();
```

Comment:
don't you still need to pass the client here?

Reply:
Yes, my bad. I thought the way it was coded was weird with the Promise wrapper, then realised after I'd changed it why it was necessary but didn't fully put it back clearly.